internal/config/add-account.go

```diff
@@ -0,0 +1,133 @@
+/*
+ * Flow CLI
+ *
+ * Copyright 2019-2021 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/onflow/flow-cli/internal/command"
+	"github.com/onflow/flow-cli/pkg/flowcli/config"
+	"github.com/onflow/flow-cli/pkg/flowcli/output"
+	"github.com/onflow/flow-cli/pkg/flowcli/project"
+	"github.com/onflow/flow-cli/pkg/flowcli/services"
+)
+
+type flagsAddAccount struct {
+	Name     string `flag:"name" info:"Name for the account"`
+	Address  string `flag:"address" info:"Account address"`
+	KeyIndex string `default:"0" flag:"key-index" info:"Account key index"`
+	SigAlgo  string `default:"ECDSA_P256" flag:"sig-algo" info:"Account key signature algorithm"`
+	HashAlgo string `default:"SHA3_256" flag:"hash-algo" info:"Account hash used for the digest"`
+	Key      string `flag:"key" info:"Account private key"`
```

Comment:
I think it might be better to be explicit here and say `private-key`, given that we use `key` on other commands to refer to a public key.